backend/src/main/java/org/eclipse/tractusx/semantics/registry/dto/ShellCollectionDto.java

```diff
@@ -33,4 +33,5 @@ public class ShellCollectionDto {
     Integer currentPage;
```

Comment:
Remove unused attributes like currentPage etc. (was for offset pag.)

Reply:
Actually lot of refactoring is required, but I did not want to touch the existing code as my PR is very near to 1000 line of code :D
So I can remove these attributes in next PR if that is fine.